scrapers/Anime-DB.yml

```diff
@@ -0,0 +1,125 @@
+name: Anime-DB
+
+sceneByName:
+  action: scrapeXPath
+  queryURL: "https://adultanime.dbsearch.net/black_curtain_redirect.php?url=%2F%2Fadultanime.dbsearch.net%2Fsearch%2F%3Fkeyword%3D{}&check=true"
+  scraper: sceneSearch
+sceneByQueryFragment:
+  action: scrapeXPath
+  queryURL: "https://adultanime.dbsearch.net/black_curtain_redirect.php?url={url}&check=true"
+  queryURLReplace:
+    url:
+      - regex: "https:"
+        with:
+      - regex: \/
+        with: "%2F"
+  scraper: sceneScraper
+sceneByURL:
+- action: scrapeXPath
+  url:
+    - adultanime.dbsearch.net
+  queryURL: "https://adultanime.dbsearch.net/black_curtain_redirect.php?url={url}&check=true"
+  queryURLReplace:
+    url:
+      - regex: "https:"
+        with:
+      - regex: \/
+        with: "%2F"
+  scraper: sceneScraper
+
+xPathScrapers:
+  sceneSearch:
+    scene:
+      Title: //div[@class="item-info"]/h4/a/text()
+      URL:
+        selector: //div[@class="item-info"]/h4/a/@href
+        postProcess:
+          - replace:
+              - regex: "^"
+                with: "https:"
+      Image:
+        selector: //section[@class="item-box"]/div[@class="item-img"]/a/img/@data-src
+        postProcess:
+          - replace:
+              - regex: "^"
+                with: "https:"
```

Comment:
do the images in the search work for you?
even when trying to access them directly i get a 403 error

Reply:
No. Without CDP it worked sometimes but loading very late. Probably something to do with the lazy loading.

Reply:
Maybe adding sleep to CDP would allow it to finish loading the images.